src/main/java/org/testng/reporters/IConfiguredReporter.java

```diff
@@ -0,0 +1,6 @@
+package org.testng.reporters;
+
+public interface IConfiguredReporter {
```

Comment:
Can we please add some java docs to this interface that explains the intent and where this is likely to be helpful to be implemented ?

Reply:
I was in the middle of adding JavaDoc, but stopped to explore how nested objects are Bean-ified.

Reply:
DONE